src/util.ts

```diff
@@ -0,0 +1,182 @@
+import { SuperJson } from '@superfaceai/one-sdk';
+import { parseDocumentId } from '@superfaceai/parser';
+import * as fs from 'fs';
+import * as glob from 'glob';
+import { promisify } from 'util';
+
+export const access = promisify(fs.access);
+const read = promisify(fs.readFile);
+
+export type PrintFn = (message: string) => void;
+
+export const EXTENSIONS = {
+  provider: 'json',
+  profile: 'supr',
+  map: 'suma',
+};
+
+export type CheckCombination = {
+  profile: { scope?: string; name: string };
+  provider: string;
+};
+
+let _superJson: SuperJson;
+
+export function loadSuperJson(): SuperJson {
+  if (!_superJson) {
+    _superJson = SuperJson.loadSync().unwrap();
+  }
+
+  return _superJson;
+}
+
+export function allProfileProviderCombinations(
+  superJson: SuperJson = loadSuperJson()
+): CheckCombination[] {
+  const profiles = superJson.normalized.profiles;
+  const checkCombinations: CheckCombination[] = [];
+
+  for (const profileId in profiles) {
+    const parseResult = parseDocumentId(profileId);
+
+    if (parseResult.kind === 'error') {
+      throw new Error(`Invalid profile id: ${parseResult.message}`);
+    }
+
+    const profile = {
+      name: parseResult.value.middle[0],
+      scope: parseResult.value.scope,
+    };
+
+    const profileSettings = profiles[profileId];
+    for (const provider in profileSettings.providers) {
+      checkCombinations.push({ profile, provider });
+    }
+  }
+
+  return checkCombinations;
+}
+
+export function normalizePath(
+  path: string,
+  superJson: SuperJson = loadSuperJson()
+): string {
+  return superJson.resolvePath(path);
+}
+
+export async function exists(path: string): Promise<boolean> {
+  try {
+    await access(path);
+  } catch (err) {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    if (err.code === 'ENOENT') {
+      return false;
+    }
+    throw err;
+  }
+
+  return true;
+}
+
+export async function readFile(path: string): Promise<string> {
+  const data = await read(path);
```

Comment:
When you call read with (path, { encoding: 'utf-8' }) it will always return string.

Reply:
changed, thanks